fleet_management_api/api_impl/controllers/car.py

```
@@ -75,8 +71,8 @@ def create_cars() -> _Response:  # noqa: E501
             ids.append(model.id)
             _log_info(f"Car (ID={model.id}, name='{model.name}') has been created.")
 
-        car_states = _post_default_car_state(tenants, ids).body
-        _post_default_car_action_state(tenants, ids).body
+        car_states = _post_default_car_state(request.tenants, ids).body
+        _post_default_car_action_state(request.tenants, ids).body
```

Ruff (0.8.2)

Found useless attribute access. Either assign it to a variable or remove it.

(B018)